src/Command/ExportAssetsCommand.php

```diff
@@ -0,0 +1,146 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Command;
+
+use App\FileLogger;
+use App\SQL\FindAssets;
+use App\SQL\FindVariations;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+/**
+ * {description}
+ *
+ * @author    Samir Boulil <samir.boulil@akeneo.com>
+ * @copyright 2020 Akeneo SAS (http://www.akeneo.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+class ExportAssetsCommand extends Command
+{
+    private const ASSETS_CSV_FILENAME = 'assets.csv';
+    private const VARIATIONS_CSV_FILENAME = 'variations.csv';
+
+    protected static $defaultName = 'app:export-assets';
+
+    /** * @var FindAssets */
+    private $findAssets;
+
+    /** * @var FindVariations */
+    private $findVariations;
+
+    public function __construct(FindAssets $findAssets, FindVariations $findVariations)
+    {
+        parent::__construct(static::$defaultName);
+        $this->findAssets = $findAssets;
+        $this->findVariations = $findVariations;
+    }
+
+    protected function configure()
+    {
+        $this
+            ->setDescription('Export assets and their variations of a 3.x PIM into CSV')
+            ->addArgument('fileDir', InputArgument::REQUIRED, 'The fileDir of the file to import.')
+        ;
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $this->io = new SymfonyStyle($input, $output);
+        $fileDir = $input->getArgument('fileDir');
+
+        $this->io->title('Export PAM assets and their variations in CSV files.');
```

Comment:
```suggestion
        $this->io->title('Export PAM assets and their variations in CSV files from old mysql tables.');
```

Reply:
I thought this was not relevant, but why not

Reply:
done